docs/02-module-block-2/00-structure.adoc

```diff
@@ -2,22 +2,11 @@
 // (c) iSAQB e.V. (https://isaqb.org)
 // ====================================================
 
-
-// tag::DE[]
-== Lerneinheit 2
-// end::DE[]
-
-// tag::EN[]
-== Lesson 2
-// end::EN[]
+== {lesson} 2
 
 include::01-duration-terms.adoc[{include_configuration}]
 
-
-=== {learning-goals}
-
-include::LZ-2-01.adoc[{include_configuration}]
-include::LZ-2-02.adoc[{include_configuration}]
+include::02-learning-goals.adoc[{include_configuration}]
```

Comment:
Much love for merging those files into a single one.

Reply:
💜